core/src/main/java/org/apache/gravitino/listener/api/event/AlterTagFailureEvent.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.gravitino.listener.api.event;
+
+import org.apache.gravitino.NameIdentifier;
+import org.apache.gravitino.tag.TagChange;
+
```

Comment:
please add `@DeveloperApi` and simple java doc to all tag failure event, please refer to `SchemaFailureEvent`